3001-4000/LIT3326Melchiz1.xml

```diff
@@ -5,10 +5,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1" xml:lang="en" type="main">First homily by Cyril of Alexandria on
-               Melchizedek</title>
-            <title xml:lang="gez" xml:id="t2"> ድርሳን፡ ዘቄርሎስ፡ ቅዱስ፡ ሊቀ፡ ጳጳሳት፡ ዘአለስክንድርያ፡ በእንተ፡ መልከ፡
+          
+            <title xml:lang="gez" xml:id="t1" type="complete" subtype="inscriptio"> ድርሳን፡ ዘቄርሎስ፡ ቅዱስ፡ ሊቀ፡ ጳጳሳት፡ ዘአለስክንድርያ፡ በእንተ፡ መልከ፡
                ጼዴቅ፡</title>
+            <title xml:lang="gez" type="normalized" corresp="#t1">Dǝrsān za-Qerǝllos qǝddus liqa ṗāṗāsāt za-ʾAlaskǝndǝrǝyā baʾǝnta Malka Ṣedeq</title>
+             <title xml:id="t2" xml:lang="en">First homily by Cyril of Alexandria on
+               Melchizedek</title>
```

Comment:
we try to refer to non-Ethiopic texts especially known in other traditions/contexts by a neutral/uniform/English title; you can translate the ethiopic title and add a `corresp` and `type ``main `as suggested
thank you!